pydis_site/templates/404.html

```diff
@@ -0,0 +1,34 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Python Discord | 404</title>
+
+    <meta charset="UTF-8">
+
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Hind:wght@400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="{% static "css/error_pages.css" %}">
+</head>
+
+<body>
+    <div class="error-box">
+        <div class="logo-box">
+            <img src="https://raw.githubusercontent.com/python-discord/branding/b67897df93e572c1576a9026eb78c785a794d226/logos/logo_banner/logo_site_banner.svg"
+                alt="Python Discord banner" />
+        </div>
+        <div class="content-box">
+            <h1>404 — not found</h1>
```

Comment:
I'm nitpicking, but in [RFC7231](https://tools.ietf.org/html/rfc7231#section-6.5.4) where this error code is defined, it's styled with capital letters, like 404 Not Found.

```suggestion
            <h1>404 — Not Found</h1>
```